client/starwhale/__init__.py

```diff
@@ -58,6 +58,8 @@
     "Text",
     "Line",
     "Point",
+    "DsInput",
```

Comment:
Replace `DatasetInput` with `DsInput`?  for the more explicated usage

Reply:
done